src/patch-path.js

```diff
@@ -0,0 +1,9 @@
+const isWindows = require('./is-windows')
+
+module.exports = patchPathEnv
+
+function patchPathEnv(env) {
+  if (isWindows() && env.Path && env.PATH) {
+    delete env.Path
```

Comment:
Could you add some comments here to explain why we're doing this because looking at this code as it is is unclear.